### PR TITLE
Add styles to emulate the format of the book

### PR DESCRIPTION
**Pull request by c-lliope**
- Move document export to top of organization div

### CODE REVIEW

File: app/assets/stylesheets/deb.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
+    content: '';
+    content: none;
+  }
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+
```
Comment (beckysiegel) [inline app/assets/stylesheets/deb.scss]: extra space

File: app/views/organizations/show.html.erb [hunk 2]
```diff
@@ -1,49 +1,117 @@
...
+            <ul id="primary-services">
+            <li>Substance Abuse Treatment </li>
+            <li>Mental Health Treatment </li>
+            <li>Health & Wellness Education </li>
+          </ul>
+        </li>
+        <!-- Additional Services -->
+        <li>
+          <ul class="chunked">
+            <li id="communication">Internet, Phone, Voicemail</li>
```
Comment (beckysiegel) [inline app/views/organizations/show.html.erb]: These list items all (lines 26-32) have ids. I'm wondering two things:
- What's the purpose of having them classified this way? Are we planning to style them differently based on type?
- If we do want to have them classified, perhaps class attributes would be more suitable?

File: app/views/organizations/show.html.erb [hunk 3]
```diff
@@ -1,49 +1,117 @@
...
+               <li id="food">Meals</li>
+               <li id="goods">Personal Care Items</li>
+               <li id="legal">Referrals</li>
+               <li id="health">Anger Management, Dual Diagnosis Treatment, Group Counseling, Individual Counseling</li>
+               <li id="transit">Vouchers</li>
+               <li id="employment">Job Readiness/Life Skills, Post-Incarceration Support</li>
+           </ul>
+        </li>
+      </ul>
+      </div>
```
Comment (beckysiegel) [inline app/views/organizations/show.html.erb]: Dedent

File: app/views/organizations/show.html.erb [hunk 4]
```diff
@@ -1,49 +1,117 @@
...
+        <!-- Accessibility -->
+        <h5>Accessibility</h5>
+        <div>Wheelchair</div>
+        <!-- Faith -->
+        <h5>Faith Based</h5>
+        <div>No</div>
+      <!-- About -->
+        <h5>About</h5>
+        <div>
+        <%= @organization.description %>
```
Comment (beckysiegel) [inline app/views/organizations/show.html.erb]: indent

File: app/views/organizations/show.html.erb [hunk 5]
```diff
@@ -1,49 +1,117 @@
...
+</div>
 
-<p>
-  <strong>Url:</strong>
-  <%= @organization.url %>
+<!-- column 2 -->
+  <div id="col2">
+  <!-- Location -->
+  <div class="border-around">
+    <div class="map" id="map-canvas" style="height: 300px;"></div>
```
Comment (beckysiegel) [inline app/views/organizations/show.html.erb]: My fault for adding this quickly originally, but height should be set in the stylesheet.

File: app/views/organizations/show.html.erb [hunk 6]
```diff
@@ -1,49 +1,117 @@
...
+          <div class="title">Director</div>
+          <div class="phone">(415) 487-6702</div>
+          <div class="email">shirley16maciel@gmail.com </div>
+        </div>
+  </div>
+  <!-- Notes -->
+  <div class="border-above">
+      <h4>Notes</h4>
+      <div class="chunked">Clients can get a referral off our website. Appointment preferred.</div>
+  </div>
```
Comment (beckysiegel) [inline app/views/organizations/show.html.erb]: indent

**Comment by beckysiegel** [comment]
Looks good to me overall, left a few comments. The biggest has to do with a bunch of ID attributes in a list.

File: app/views/organizations/show.html.erb (hunk 2 above)
Comment (c-lliope) [inline app/views/organizations/show.html.erb]: These were all in the markup that Deb gave me - I think that based on Steve's feedback, we're going to get rid of this section anyway.

And I agree about `id`s vs `class`es. Switching all of them over now.

**Pull request merged**